qdrant-landing/content/documentation/frameworks/langchain4j.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Langchain4J
+weight: 2100
+---
+
+# Langchain4J
+
+[Langchain4J](https://github.com/langchain4j/langchain4j) is a community port of [Langchain](https://www.langchain.com/) for building context-aware AI applications in Java
+
+You can use Qdrant as a vectorstore in Langchain4J through the [`langchain4j-qdrant`](https://central.sonatype.com/artifact/dev.langchain4j/langchain4j-qdrant) module.
+
+## Setup
+
+Add the `langchain4j-qdrant` to your project dependencies.
+
+```xml
+<dependency>
+    <groupId>dev.langchain4j</groupId>
+    <artifactId>langchain4j-qdrant</artifactId>
+    <version>VERSION</version>
+</dependency>
+```
+
+## Usage
+
```

Comment:
Similar to the other file:

I think you should highlight the variable code, to make sure readers know to substitute their configuration first. Here's how I might do this:

```suggestion

Before you use the following code sample, customize the following values for your configuration:

- `YOUR_COLLECTION_NAME`: Use our [Collections](/documentation/concepts/collections) guide to create or
  list collections.
- `YOUR_HOST_URL`: Use the REST URL for your system. If you used the [Quick Start](/documentation/quick-start/) guide,
  it may be http://localhost:6333. If you've deployed in the [Qdrant Cloud](/documentation/cloud/), you may have a
  longer URL such as `https://example.location.cloud.qdrant.io`.
- `YOUR_API_KEY`: Substitute the API key associated with your configuration.

Reply:
Since Langchain4J uses the Qdrant Java client through the GRPC interface.
https://github.com/qdrant/landing_page/pull/571/commits/898d18922eaa1f2770e47cc3a64ac54cf6206129